Kill resize-ing and zero-ing from codegen.

gchanan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #38073 Move (most) generated return statements for TH functions out of the switch.
* #38072 Remove codegen for IntArrayRefStride, which isn't used.
* #38071 Macro generate ScalarTypeToCPPType, including all ScalarTypes.
* **#37958 Kill resize-ing and zero-ing from codegen.**

All codegen invocations have been removed at this point, so this has no effect.

Differential Revision: [D21433215](https://our.internmc.facebook.com/intern/diff/D21433215)

gchanan committed:
Kill resize-ing and zero-ing from codegen.

All codegen invocations have been removed at this point, so this has no effect.

[ghstack-poisoned]

bhosmer commented on aten/src/ATen/nn_parse.py:
```diff
@@ -294,15 +294,6 @@ def initialize_output_arg(arg):
-        if arg['name'] == 'grad_weight' and base['name'] != '_thnn_conv2d' and base['name'] != '_thnn_conv_depthwise2d':
-            arg['resize'] = 'weight'
-            arg['zero'] = True
-        if arg['name'] == 'grad_bias' and base['name'] != '_thnn_conv2d' and base['name'] != '_thnn_conv_depthwise2d':
```
hahahahah NEVER MIND EARLIER COMMENTS

gchanan committed:
Update on "Kill resize-ing and zero-ing from codegen."


All codegen invocations have been removed at this point, so this has no effect.

Differential Revision: [D21433215](https://our.internmc.facebook.com/intern/diff/D21433215)

[ghstack-poisoned]

gchanan committed:
Update on "Kill resize-ing and zero-ing from codegen."


All codegen invocations have been removed at this point, so this has no effect.

Differential Revision: [D21433215](https://our.internmc.facebook.com/intern/diff/D21433215)

[ghstack-poisoned]

gchanan committed:
Update on "Kill resize-ing and zero-ing from codegen."


All codegen invocations have been removed at this point, so this has no effect.

Differential Revision: [D21433215](https://our.internmc.facebook.com/intern/diff/D21433215)

[ghstack-poisoned]

gchanan committed:
Update on "Kill resize-ing and zero-ing from codegen."


All codegen invocations have been removed at this point, so this has no effect.

Differential Revision: [D21433215](https://our.internmc.facebook.com/intern/diff/D21433215)

[ghstack-poisoned]

gchanan committed:
Update on "Kill resize-ing and zero-ing from codegen."


All codegen invocations have been removed at this point, so this has no effect.

Differential Revision: [D21433215](https://our.internmc.facebook.com/intern/diff/D21433215)

[ghstack-poisoned]
